Vagrantfile

```diff
@@ -56,31 +54,42 @@ $bootstrap = <<BOOTSTRAP
   port=3306" >> /etc/my.cnf
   sudo chown -R mysql /var/lib/mysql/
   sudo /etc/init.d/mysql start
-  #service mysql restart
-  mysql -u root -proot -e "CREATE USER 'django_user' IDENTIFIED BY 'secret'"
-  mysql -u root -proot -e "DROP DATABASE IF EXISTS gbe_dev"
-  mysql -u root -proot  -e "CREATE DATABASE gbe_dev"
-  mysql -u root -proot  -e "USE gbe_dev"
-  mysql -u root -proot  -e "GRANT ALL ON gbe_dev.* to 'django_user'@'%' IDENTIFIED BY 'secret' WITH GRANT OPTION"
-  mysql -u root -proot  -e "GRANT ALL ON gbe_dev.* to 'django_user'@'%' IDENTIFIED BY 'secret'"
-  mysql -u root -proot  -e "GRANT ALL ON test_gbe_dev.* to 'django_user'@'%' IDENTIFIED BY 'secret'"
-  mysql -u root -proot  -e "flush privileges"
  
   echo "your initialization shell scripts go here"
   sudo apt-fast -y install python-dev
   sudo apt-fast -y install python-pip
   sudo apt-fast -y install libjpeg-dev
-  sudo apt-fast -y  install libjpeg8-dev
+  sudo apt-fast -y install libjpeg8-dev
   sudo apt-fast -y install libpng3 
   sudo apt-fast -y install libfreetype6-dev
+  sudo apt-fast -y install gettext
   sudo pip install --requirement /vagrant/config/requirements.txt
-  cp /vagrant/aliases /vagrant/dbreset /home/vagrant/
+  ifmkdir /vagrant/tmp; cd /vagrant/tmp
+  ifmkdir /vagrant/static; ifmkdir /vagrant/expo/logs; ifmkdir /vagrant/media
+  cp /vagrant/aliases /vagrant/dbreset /home/vagrant
   chown -R vagrant:vagrant /home/vagrant
-  echo "source /home/vagrant/aliases" >> /home/vagrant/.bashrc
-  ifmkdir /vagrant/expo/logs; ifmkdir /vagrant/uploads
-  ifmkdir /vagrant/uploads/images; ifmkdir /vagrant/uploads/images/fullsize
-  ifmkdir /vagrant/uploads/images/mini; ifmkdir /vagrant/uploads/images/thumb
-  chown -R vagrant:vagrant /vagrant
+  echo "source /home/vagrant/aliases" >> /home/vagrant/.bashrc 
+
+###  The next four lines will have to be done by hand, until we get a good test DB setup 
+###  cat /vagrant/config/gbe2016test_data.sql | python manage.py dbshell
+###  python manage.py syncdb --all
+###  python manage.py migrate --all
+###  python manage.py collectstatic
+
+  if [ -f /vagrant/config/local_settings.py ]
+  then
+      cp /vagrant/config/local_settings.py /vagrant/expo/expo
+  fi
+  if [ -f /vagrant/config/gbe_*.sql ]
```

Comment:
This seems to assume a naming convention that is not like the one you use on the live server, meaning we have to remember to rename database dumps when we take them down from the server. Could we get these in sync?